templates/apps/wry/src/lib.rs.hbs

```diff
@@ -1,21 +1,15 @@
-use anyhow::Result;
-#[cfg(target_os = "android")]
-use wry::android_binding;
-use wry::{
-    application::{
-        event::{Event, StartCause, WindowEvent},
-        event_loop::{ControlFlow, EventLoop, EventLoopWindowTarget},
-        window::WindowBuilder,
-    },
-    http::Response,
-    webview::{WebView, WebViewBuilder},
+use tao::{
+    event::{Event, StartCause, WindowEvent},
+    event_loop::{ControlFlow, EventLoop, EventLoopWindowTarget},
+    window::WindowBuilder,
 };
+use wry::{http, WebView, WebViewBuilder};
 
 #[cfg(target_os = "android")]
 fn init_logging() {
     android_logger::init_once(
         android_logger::Config::default()
-            .with_min_level(log::Level::Trace)
+            .with_max_level(log::LevelFilter::Trace)
```

Comment:
Is it never interesting for users to get a higher level of logging?

Reply:
I think the behavior is the same, it is just the `with_min_level` doesn't exist anymore